docs/csharp/language-reference/keywords/using-directive.md

```diff
@@ -50,46 +92,51 @@ class Program
+
+In the example, the `using static` directive could also have been applied to the <xref:System.Double> type. Adding that directive would make it possible to call the <xref:System.Double.TryParse(System.String,System.Double@)> method without specifying a type name. However, using `TryParse` without a type name creates less readable code, since it becomes necessary to check the `using static` directives to determine which numeric type's `TryParse` method is called.
+
+## using alias
+
+Create a `using` alias directive to make it easier to qualify an identifier to a namespace or type. In any `using` directive, the fully qualified namespace or type must be used regardless of the `using` directives that come before it. No `using` alias can be used in the declaration of a `using` directive. For example, the following example generates a compiler error:
 
 ```csharp
 using s = System.Text;
 using s.RegularExpressions; // Generates a compiler error.
 ```
 
-Create a `using` directive to use the types in a namespace without having to specify the namespace. A `using` directive does not give you access to any namespaces that are nested in the namespace you specify.
-
-Namespaces come in two categories: user-defined and system-defined. User-defined namespaces are namespaces defined in your code. For a list of the system-defined namespaces, see [.NET API Browser](../../../../api/index.md).
-
-## Example 1
-
 The following example shows how to define and use a `using` alias for a namespace:
 
-[!code-csharp[csrefKeywordsNamespace#8](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsNamespace/CS/csrefKeywordsNamespace2.cs#8)]
+:::code language="csharp" source="./snippets/csrefKeywordsNamespace2.cs" id="Snippet8":::
 
-A using alias directive cannot have an open generic type on the right hand side. For example, you cannot create a using alias for a `List<T>`, but you can create one for a `List<int>`.
-
-## Example 2
+A using alias directive can't have an open generic type on the right-hand side. For example, you can't create a using alias for a `List<T>`, but you can create one for a `List<int>`.
 
 The following example shows how to define a `using` directive and a `using` alias for a class:
 
-[!code-csharp[csrefKeywordsNamespace#9](~/samples/snippets/csharp/VS_Snippets_VBCSharp/csrefKeywordsNamespace/CS/csrefKeywordsNamespace2.cs#9)]
+:::code language="csharp" source="./snippets/csrefKeywordsNamespace2.cs" id="Snippet9":::
 
 ## C# language specification
 
 For more information, see [Using directives](~/_csharplang/spec/namespaces.md#using-directives) in the [C# Language Specification](/dotnet/csharp/language-reference/language-specification/introduction). The language specification is the definitive source for C# syntax and usage.
 
+For more information on the *global using* modifier, see the [global usings feature specification - C# 10.0](~/_csharplang/proposals/csharp-10.0/GlobalUsingDirective.md).
```

Comment:
Can you use a global static alias? I'm a bit curious now about some of the flexibility and limitations.